zulip_bots/zulip_bots/bots/followup/test_followup.py

```diff
@@ -7,8 +7,10 @@
 
 class TestFollowUpBot(BotTestCase):
     bot_name = "followup"
+    mock_config_default_stream = {'stream': 'followup'}
```

Comment:
I'm not sure having these as class variables is necessary.  You can probably define them directly in the tests.  And it seems like "followup" **is** the default, so you should be mocking config_info to return None or something, right?